service_management/azure/test/integration/service_bus/queues_scenario_test.rb

```diff
@@ -13,18 +13,36 @@
 # limitations under the License.
 #--------------------------------------------------------------------------
 require 'integration/test_helper'
+require 'integration/service_bus/scenario_helper'
 
-describe 'ServiceBus Queues Scenario' do
-  let(:queue_name) { ServiceBusQueueNameHelper.name }
-
+describe Azure::ServiceBus::ServiceBusService do
   subject { Azure::ServiceBus::ServiceBusService.new }
-  after { ServiceBusQueueNameHelper.clean }
+  let(:queue_name) { 'test-queue' }
+
+  describe 'Service bus queue scenario' do
+    before do
+      VCR.insert_cassette "service_bus/#{name}"
+    end
+
+    after do
+      subject.delete_queue queue_name
+      VCR.eject_cassette
+    end
+
+    it 'should upload many messages and read them back' do
```

Comment:
Since creation of the queue is inside the test, should we have deletion inside the test instead of the 'after' block? (with only 1 test it doesn't matter too much, but for consistency).